### PR TITLE
added an about page that has some info about the group

### PR DESCRIPTION
**Pull request by deej-tsn**
image

### CODE REVIEW

File: src/pages/about-us.astro [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                With projects, exercises, and an active global network, it’s a great place to start (or advance) your coding journey. And with chapters like freeCodeCamp London, there are opportunities to connect locally too!
+                At freeCodeCamp London, we host regular meetups to bring our community together.
+                Whether you’re working through coding challenges, looking for career advice, or just interested in meeting fellow coders, our meetups are designed to be relaxed and beginner-friendly.
+            </p>
+            Here’s what you can expect:
+            <ul class="mt-0 list-disc pl-4">
+    
+                <li>Group Learning: At the start, we usually split into groups based on interests. You might join a group tackling specific coding challenges, discussing job-seeking strategies, or chatting about all things tech.</li>
+    
+                <li>Show and Tell: We start each meetup with a short “show and tell” where members can share projects, recent learnings, or anything coding-related. Interested? Let us know on Slack!</li>
```
Comment (webtechalex) [inline src/pages/about-us.astro]: We don't use slack. We use Discord! 😁

File: src/pages/about-us.astro [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+import FAQCard from "../components/FAQCard.astro";
+import { faqAnswer } from "../data/faq";
+import Layout from "../layouts/Layout.astro";
+---
+
+<Layout title="About Us">
+    <div class="w-full flex items-center text-left">
```
Comment (webtechalex) [inline src/pages/about-us.astro]: We are getting a lot of low-level markup in our page-level components. Not necessarily for this PR, but I wonder if we can do a better job of composing UI components to make the pages a bit more readable. I may have some time this week to try to figure that out

**Comment by deej-tsn** [comment]
I have made all the recommended changes to the pull request under a new commit, not sure if this is the way to adjust the pull request but feel free to check it out

**Comment by webtechalex** [comment]
Indeed, it's best to respond to any comments just by adding commits to the branch and pushing again to remote. Only thing is that now, this PR does more than it says on the tin. Happy to merge as this is good work going in the right direction, (also it's my fault for introducing a side-tracking comment and not being clear that I didn't want you to respond to it here!) but it's better practice to stick to one task per PR so we should be careful about that in future contributions

**Pull request merged**